### PR TITLE
Slimernalia is good now, actually

### PR DESCRIPTION
**Pull request by C-rank**
Patch Notes:
- There's now a cap to the amount of festivity you can gain per gift! It's still very high, just not horrifying!
- Slimernalia kingpins now get their festivity reset upon election! Stay competitive, or be banished from the auditorium!
- Juveniles can now be !scout'ed while Slimernalia is active
- All this festivity has had unforeseen side effects... A certain slime deity has manifested, and has something to say at 6am!
- Fixed some gifting bugs

Developer Notes:
- Added the !setfestivity command, so now we can do hot changes without touching the database
- Removed a bunch of vestigial print statements from around the code
- Deduplicated a bunch of item definitions
- ENDLESS WAR now ignores suggestion-box and splatify completely

### CODE REVIEW

File: ew/utils/frontend.py [hunk 1]
```diff
@@ -496,14 +496,31 @@ async def update_slimernalia_kingpin(client, server):
     # Update the new kingpin of slimernalia
     new_kingpin = EwUser(id_user=ewutils.get_most_festive(server), id_server=server.id)
     new_kingpin.slimernalia_kingpin = True
+    
+    # Reset the new kingpin's festivity upon getting the award
+    old_festivity = new_kingpin.festivity
```
Comment (aristotlepenguin) [inline ew/utils/frontend.py:501]: the current term is 6 hours, it triggers at the same time as leaderboards. are you sure we should be resetting it that quickly?

File: ew/cmd/cmds/cmdcmds.py [hunk 2]
```diff
@@ -2092,7 +2092,7 @@ async def wrap(cmd):
             response = "It's a nice gesture, but trying to gift someone a Soulbound item is going a bit too far, don't you think?"
         # Check if the item has been gifted before
         elif item.item_props.get('gifted') == "true":
-            print("I am going to kill myself")
+            # print("I am going to kill myself")
```
Comment (aristotlepenguin) [inline ew/cmd/cmds/cmdcmds.py:2095]: let's be real, we're not going to use this later it can just be deleted

File: ew/static/cfg.py [hunk 3]
```diff
@@ -41,6 +41,7 @@
 #sure, but is slimernalia active?
 slimernalia_active = True
 
+forbidden_channels = ["suggestion-box", "splatify"]
```
Comment (aristotlepenguin) [inline ew/static/cfg.py:44]: Splatify is actually a subzone in crookline too, so we can't prevent command responses there. Suggestion box isn't either, but whether you leave it in is up to you

**Review by aristotlepenguin — CHANGES_REQUESTED** [review]

File: ew/utils/frontend.py (hunk 1 above)
Comment (C-rank) [reply ew/utils/frontend.py:501]: we'll test the 6 hour period live on the test bot, if the community finds it too harsh or stifling we can always move to a 12 hour period

**Pull request merged**